src/Kiota.Builder/Refiners/PhpRefiner.cs

```diff
@@ -135,11 +135,13 @@ public override Task Refine(CodeNamespace generatedCode, CancellationToken cance
         new (x => x is CodeProperty prop && prop.IsOfKind(CodePropertyKind.RequestAdapter),
             "Microsoft\\Kiota\\Abstractions", "RequestAdapter"),
         new (x => x is CodeMethod method && method.IsOfKind(CodeMethodKind.RequestGenerator),
-            "Microsoft\\Kiota\\Abstractions", "HttpMethod", "RequestInformation", "RequestOption"),
+            "Microsoft\\Kiota\\Abstractions", "HttpMethod", "RequestInformation"),
```

Comment:
can you share some context on this removal?

Reply:
Request option was moved to the Request configuration class(Which exists in it's own file), meaning we no longer need the import in request builder classes.